app/user/layout.tsx

```diff
@@ -0,0 +1,18 @@
+import SideMenu from '@/components/SideMenu';
+import { ReactNode } from 'react';
+
+export default function ProfilePageLayout({
+  children,
+}: Readonly<{
+  children: ReactNode;
+}>) {
+  return (
+    <div className='flex h-screen '>
+      <SideMenu />
```

Comment:
move SideMenu to dashboard layout because it will be used across all dashboard pages